doc/security/implementation_guidelines/README.md

```diff
@@ -1 +1,5 @@
 # Implementation Guidelines
+
+## [Secure Hardware Design Guidelines](./hardware/README.md)
+
+## [Reset vs. Non-Reset Flops](./reset_vs_non-reset_flops/README.md)
```

Comment:
Nit: I think you've capitalized this as "non-reset" elsewhere. I probably prefer that version, but don't feel strongly. But it's probably better to be consistent :-)

Reply:
I agree that this may not look consistent but I think it actually is. :-)
I tried to follow the currently used style in our documentation. My understanding is that:
- for titles (#) and section (##) we capitalize words, whereas
- for sub-sections (###) we just capitalize the first word

One exception to this rule is the `## References and further reading` section below.